library/games/Senet/0.json

```diff
@@ -0,0 +1,1346 @@
+    },
+    "image": "/assets/-2115246459_536956",
+    "x": -96,
+    "z": 1
+  },
+  "stick2": {
+    "id": "stick2",
+    "x": 743,
+    "y": 725,
+    "width": 40,
+    "height": 200,
+    "borderRadius": 15,
+    "z": 1,
+    "css": {
+      "background": "#444",
+      "border": "3px solid #fff",
+      "rotation": 90,
+      "transition": "linear 0.2s"
+    },
+    "faces": [
+      {
+        "css": {
+          "background": "#444",
+          "border": "3px solid #fff",
+          "rotation": 90,
+          "transition": "linear 0.2s"
+        }
+      },
+      {
+        "css": {
+          "background": "#fff",
+          "border": "3px solid #444",
+          "transition": "linear 0.2s"
+        }
+      }
+    ],
+    "resetProperties": {
+      "rotation": null,
+      "activeFace": 1
+    },
+    "activeFace": 1
+  },
+  "stick3": {
+    "id": "stick3",
+    "x": 807,
+    "y": 725,
+    "width": 40,
+    "height": 200,
+    "borderRadius": 15,
+    "z": 1,
+    "css": {
+      "background": "#444",
+      "border": "3px solid #fff",
+      "rotation": 90,
+      "transition": "linear 0.2s"
+    },
+    "faces": [
+      {
+        "css": {
+          "background": "#444",
+          "border": "3px solid #fff",
+          "rotation": 90,
+          "transition": "linear 0.2s"
+        }
+      },
+      {
+        "css": {
+          "background": "#fff",
+          "border": "3px solid #444",
+          "transition": "linear 0.2s"
+        }
+      }
+    ],
+    "resetProperties": {
+      "rotation": null,
+      "activeFace": 0
+    }
+  },
+  "stick4": {
+    "id": "stick4",
+    "x": 870,
+    "y": 725,
+    "width": 40,
+    "height": 200,
+    "borderRadius": 15,
+    "z": 1,
+    "css": {
+      "background": "#444",
+      "border": "3px solid #fff",
+      "rotation": 90,
+      "transition": "linear 0.2s"
+    },
+    "faces": [
+      {
+        "css": {
+          "background": "#444",
+          "border": "3px solid #fff",
+          "rotation": 90,
+          "transition": "linear 0.2s"
+        }
+      },
+      {
+        "css": {
+          "background": "#fff",
+          "border": "3px solid #444",
+          "transition": "linear 0.2s"
+        }
+      }
+    ],
+    "resetProperties": {
+      "rotation": null,
+      "activeFace": 1
+    },
+    "activeFace": 1
+  },
+  "clicker": {
+    "id": "clicker",
+    "x": 560,
+    "y": 701,
+    "width": 480,
+    "height": 250,
+    "z": 1,
+    "movable": false,
+    "movableInEdit": false,
+    "layer": 2,
+    "clickRoutine": [
+      {
+        "func": "FOREACH",
+        "in": [
+          "stick1",
+          "stick2",
+          "stick3",
+          "stick4"
+        ],
+        "loopRoutine": [
+          "var randomClick = randInt 1 2",
+          "var randomRot = randInt 0 359",
+          {
+            "func": "CLICK",
+            "collection": [
+              "${value}"
+            ],
+            "count": "${randomClick}"
+          },
```

Comment:
"count": "${randomClick}" should have no effect. only the final face will be displayed

Reply:
The count is necessary in order to click the face once to change the color or twice to leave it as is.